debian/kafka/Dockerfile

```diff
@@ -29,12 +29,15 @@ ENV KAFKA_ZOOKEEPER_CONNECT=${KAFKA_ZOOKEEPER_CONNECT}
 ARG KAFKA_ADVERTISED_LISTENERS
 ENV KAFKA_ADVERTISED_LISTENERS=${KAFKA_ADVERTISED_LISTENERS}
 
+ARG RELEASE_QUALITY=${RELEASE_QUALITY}
+
 ENV COMPONENT=kafka
 
 # primary
 EXPOSE 9092
 
 RUN echo "===> installing ${COMPONENT}..." \
+    && if [ "${RELEASE_QUALITY}" = "beta" ]; then CONFLUENT_PLATFORM_LABEL=""; fi \
```

Comment:
I still don't get why we need to set this to empty for Kafka only as the _LABEL should have the -betaNNNN part and we do build Kafka with the same versioning scheme as other CP components as Kafka is part of CP? 

Looking at our deb package's version, it looks like we are not packaging Kafka with the correct version and we should fix that instead? @aayars @ewencp to double check.

hub is correct, but kafka is not (missing ~betaNNNN...)
----------------------------------------------------------
https://s3.console.aws.amazon.com/s3/object/jenkins-confluent-packages/v5.1.0-beta201806200051/1/deb/5.1/dists/stable/main/binary-amd64/Packages?region=us-west-2&tab=overview
```Package: confluent-hub-client
Priority: optional
Section: misc
Installed-Size: 12006
Maintainer: Confluent Packaging <packages@confluent.io>
Architecture: all
Version: 5.1.0~beta201806200051-1
Depends: confluent-common
Filename: pool/main/c/confluent-hub-client/confluent-hub-client_5.1.0~beta201806200051-1_all.deb
Size: 9910074
MD5sum: 4c62c569ee687bc50f90ec1e751f4930
SHA1: b9df42fc02de186661301771239130b875f5aff6
SHA256: c84a7f3597ba8328df2aa99a1970721b548ecd4b0430ae2feaadfd0bf5624f81
SHA512: 66bbcf6e9ba74bcfbe32382d3cf51a1ad957ba40b87e7af7bdcac7d596e5007d8819a121a25db74b8c60544ac02cb2bfb00bdd9edad8abd5a2d65e5b2efdc08c
Description: Confluent Hub client.
Homepage: http://confluent.io

Package: confluent-kafka-2.11
Priority: optional
Section: misc
Installed-Size: 64652
Maintainer: Confluent Packaging <packages@confluent.io>
Architecture: all
Source: confluent-kafka
Version: 2.1.0-1
```

Refs:
1. https://github.com/confluentinc/cp-docker-images-overlay/pull/3/files#diff-3ef4bcfef601b562953c567a2e9bb586R38
2. https://s3-us-west-2.amazonaws.com/jenkins-confluent-packages-maven/v5.1.0-beta201806200051/1/maven/org/apache/kafka/kafka_2.12/2.1.0-beta201806200051/kafka_2.12-2.1.0-beta201806200051.jar

Reply:
Will look into this and get back .. will be validating this code again as I address the feedback comment.